tools/clang/test/lit.cfg

```diff
@@ -475,6 +510,12 @@ if config.enable_backtrace == "1":
 if lit_config.params.get("run_long_tests", None) == "true":
     config.available_features.add("long_tests")
 
+# Check if spirv is enabled.
+if config.spirv:
+    config.available_features.add("spirv")
+else:
+    config.available_features.add("nospirv")
```

Comment:
I don't see where we're using the feature here, but I'm also not sure we need an inverse of the feature. Does `UNSUPPORTED` not cover the use case?

Reply:
Removed.